notebooks/en/clean_dataset_judges_distilabel.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<p>I would also put the link on the dataset name instead of the text "Hugging Face Hub"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>